projects/tuist/features/dependencies.feature

```diff
@@ -15,3 +15,11 @@ Feature: Tuist dependencies.
         Then tuist fetches dependencies
         Then tuist generates the project
         Then tuist builds the scheme App from the project
+
+    Scenario: The project is a sub project within a workspace with SPM Dependencies.swift (app_with_spm_dependencies)
+        Given that tuist is available
+        And I have a working directory
+        Then I copy the fixture app_with_spm_dependencies into the working directory
+        Then tuist fetches dependencies
+        Then tuist generates the project at /Features/FeatureOne
+        Then tuist builds the scheme FeatureOneFramework from the project    
```

Comment:
Shouldn't this be
```suggestion
        Then tuist builds the scheme App from the project

    Scenario: The project is a sub project within a workspace with SPM Dependencies.swift (app_with_spm_dependencies)
        Given that tuist is available
        And I have a working directory
        Then I copy the fixture app_with_spm_dependencies into the working directory
        Then tuist fetches dependencies
        Then tuist generates the project at /Features/FeatureOne
        Then tuist builds the scheme FeatureOneFramework from the project at Features/FeatureOne
```
or something similar?

Reply:
I have updated the cucumber description and also added the respective step to the `tuist.rb` file